package.json

```diff
@@ -76,7 +76,7 @@
     "ncp": "^2.0.0",
     "object-assign": "^4.1.1",
     "platform": "^1.1.0",
-    "prettier": "^1.2.2",
+    "prettier": "1.2.2",
```

Comment:
Ah, fair point. 

👍 I'm in favor of this. I can see it being annoying to randomly write a bunch of changes.